src/ResourceManager/Network/Commands.Network/ApplicationGateway/TrustedRootCertificate/AddAzureApplicationGatewayTrustedRootCertificateCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Add, "AzureRmApplicationGatewayTrustedRootCertificate", SupportsShouldProcess = true), 
```

Comment:
Kiindly use `ResourceManager.Common.AzureRMConstants.AzureRMPrefix` for the AzureRm part of the prefix to the noun part. Example [here in the current PR itself](https://github.com/Azure/azure-powershell/pull/7121/files#diff-f36fcbd8e47bf79727d50081d50419e6R21). This applies to all instances where the name of the cmdlet is defined in the PR.